app/iothub_client_sample_mqtt.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "AzureIotHub.h"
+#include "Arduino.h"
+#include "config.h"
+#include "utility.h"
+#include "iothub_client_sample_mqtt.h"
+#include <ArduinoJson.h>
+
+static IOTHUB_CLIENT_LL_HANDLE iotHubClientHandle;
+static bool messagePending = false;
+static bool messageSending = true;
+
+void executeCommand(const char *jsonCommand)
+{
+    StaticJsonBuffer<MESSAGE_MAX_LEN> jsonBuffer;
+    JsonObject& root = jsonBuffer.parseObject(jsonCommand);
+    if (!root.success())
+    {
+        LogInfo("Parse command %s failed", jsonCommand);
+        return;
+    }
+
+    if(!root.containsKey("command") || !root["command"].is<const char *>())
+    {
+        LogInfo("No command string value found in %s", jsonCommand);
+        return;
+    }
+
+    const char * command = root["command"];
+    if(strcmp(command, "blink") == 0)
+    {
+        blinkLED();
+    }
+    else if(strcmp(command, "start") == 0)
+    {
+        messageSending = true;
+    }
+    else if(strcmp(command, "stop") == 0)
+    {
+        messageSending = false;
+    }
+}
+
+static IOTHUBMESSAGE_DISPOSITION_RESULT c2dMessageCallback(IOTHUB_MESSAGE_HANDLE message, void *userContextCallback)
+{
+    const char *buffer;
+    size_t size;
+
+    if (IoTHubMessage_GetByteArray(message, (const unsigned char **)&buffer, &size) != IOTHUB_MESSAGE_OK)
+    {
+        LogInfo("unable to IoTHubMessage_GetByteArray");
+        return IOTHUBMESSAGE_REJECTED;
+    }
+    else
+    {
+        char *temp = (char *)malloc(size + 1);
+        if (temp == NULL)
+        {
+            LogInfo("Failed to malloc for command");
+            return IOTHUBMESSAGE_REJECTED;
+        }
+        memcpy(temp, buffer, size);
+        temp[size] = '\0';
+        LogInfo("Receive C2D message: %s", temp);
+        executeCommand(temp);
```

Comment:
No needed, remove executeCommand(),  just call blinkLED();, refer to https://github.com/Azure-Samples/iot-hub-feather-huzzah-client-app/blob/master/app/iothubClient.ino#L102